### DISCUSSION TITLE
How the hell do I enable Server Actions in my project

### DISCUSSION THREAD

**Discussion by AlizerUncaged** [vercel/next.js#58636] [Help] [answered]
### Summary

I get the error below telling me to _"please enable the feature flag in your Next.js config"_ but I've been googling for an hour now and I have no idea how to do this, any help? 

![image](https://github.com/vercel/next.js/assets/86959368/c14ac93e-2e63-4085-b4e5-6ed0e8940650)

It would be very helpful if the docs at https://nextjs.org/docs/app/api-reference/functions/server-actions say something about this

**Comment by icyJoseph** [comment, marked as answer]
Hi,

It is as simple as, in your `next.config.js`:

```js
/** @type {import('next').NextConfig} */
const config = {
  experimental: {
    serverActions: true,
  },
};

module.exports = config;

```

**Reply by icyJoseph** [reply]
@AlizerUncaged i wonder what Next.js version do you have? I have a feeling that in 14, the flag's not needed anymore, which is why you can't easily find it in the docs.

**Reply by bugplanter** [reply]
The solution given works in 13, but is not needed in 14. You will get the `"Invalid next.config.js options detected"` as the feature is now out of experimental.

**Comment by tdombui** [comment]
As already mentioned, you'll get the following error:
 ⚠ Invalid next.config.js options detected:
 ⚠     Expected object, received boolean at "experimental.serverActions"
 ⚠ See more info here: https://nextjs.org/docs/messages/invalid-next-config
 ⚠ Server Actions are available by default now, `experimental.serverActions` option can be safely removed.

**Comment by JMoralesWMP** [comment]
It worked for me to clear cookies after setting the next.config file

**Reply by AdamkoXD** [reply]
how
